src/buf_encoder.rs

```diff
@@ -10,10 +10,12 @@
 
 use core::borrow::Borrow;
 
-pub use out_bytes::OutBytes;
-
 use super::Case;
 
+#[rustfmt::skip]                // Keep public re-exports separate.
+#[doc(inline)]
+pub use self::out_bytes::OutBytes;
```

Comment:
Its in a private module, without this it would be private to the crate - is that what we want? 

Note, changing it is not in scope for this PR so we can still merge.

Reply:
:man_facepalming: sorry, I thought it's in lib.rs